src/main/java/mate/model/Driver.java

```diff
@@ -54,4 +58,20 @@ public boolean equals(Object o) {
     public int hashCode() {
         return Objects.hash(id, name, licenseNumber);
     }
+
```

Comment:
Do we need to include login into equals and hashCode?

Reply:
Fixed. Should I use here sensitive info like password or something?

Reply:
You definitely shouldn't include the password in the toString(), and I don't think that you have to include password to equals and hashCode (but this is not very important in this case).